src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -66,7 +68,8 @@ public ComputeEngineInstance(
       RetentionStrategy retentionStrategy,
       Integer launchTimeout,
       // NOTE(craigatgoogle): Could not use Optional due to serialization req.
-      @Nullable String javaExecPath)
+      @Nullable String javaExecPath,
+      @Nullable GoogleKeyPair sshKeyPair)
```

Comment:
This constructor starts looking as most constructors in this project :0
Can we try doing something different here? Or we are waiting for lombok?
Maybe we should split it somehow.. Cause its waaay to long..

Reply:
In my lombok changes, this constructor will be kept around but made private, then I add the `@Builder` annotation on the constructor to create a builder. This is needed because lombok does not include superclass fields unless you use the experimental `@SuperBuilder`. That would require all superclasses to also be annotated with `@SuperBuilder` which is something we don't have any control over.

Reply:
I agree these constructors are too long. I've been holding off digging into the re-factoring in anticipation of the PR @stephenashank is working on.

Reply:
Ok..

Reply:
One way to cut down on the constructor size is to switch from using final fields to using non-final fields with setters, then using a custom builder that uses those setters.

Reply:
Or we should split that instance into some subcomponents..

Reply:
But I just wanted to comment it here.. It should not be part of this PR..